registrations/views.py

```diff
@@ -134,3 +138,19 @@ def get(self, request):
             )
             pending_user.delete()
             return HttpResponse(render_to_string('registration/validation-success.html'))
+
+class UserExternalTokenViewset(viewsets.ModelViewSet):
+    queryset = UserExternalToken.objects.exclude(is_disabled=True)
```

Comment:
We can show all the user tokens

```suggestion
    queryset = UserExternalToken.objects.exclude(is_disabled=True)
```